src/components/SignInForm/SignInForm.js

```diff
@@ -1,19 +1,61 @@
+import { useState } from "react";
+
 import classes from "./SignInForm.module.css";
+import Input from "../Input/Input";
+
+const emailValidation = (email) => /\w+@\w+\.\w+/.test(email);
+
+const passwordValidation = (pass) =>
+  pass.length >= 8 && /\w/.test(pass) && /\d/.test(pass);
+
+const INPUTS_COUNT = 2;
 
 const SignInForm = () => {
+  const [validInputsIds, setValidInputsIds] = useState([]);
+
+  const changeValidInputsIds = ({ typeOfChange, inputId }) => {
+    const idIsAlreadyContained = validInputsIds.includes(inputId);
+
+    if (typeOfChange === "add" && !idIsAlreadyContained) {
+      let newState = validInputsIds.slice();
+      newState.push(inputId);
+      setValidInputsIds(newState);
+    } else if (typeOfChange === "remove" && idIsAlreadyContained) {
+      let newState = validInputsIds.filter((someId) => someId !== inputId);
+      setValidInputsIds(newState);
+    }
```

Comment:
Could you please refactor this logic to not use INPUTS_COUNT?
(ex. Using array of objects)
Or you can explain, if you think your approach is better